content/custom_resources/about.md

```diff
@@ -86,20 +76,24 @@ action :delete do
   package 'httpd' do
     action :remove
   end
+
+  file '/var/www/html/index.html' do
+    action :delete
+  end
 end
 ```
 
 where
 
--   `homepage` is a property that sets the default HTML for the
-    `index.html` file with a default value of `'<h1>Hello world!</h1>'`
--   the `action` block uses the built-in collection of resources to tell
-    Chef Infra Client how to install Apache, start the service, and then
-    create the contents of the file located at
-    `/var/www/html/index.html`
--   `action :create` is the default resource, because it is listed
-    first; `action :delete` must be called specifically (because it is
-    not the default action)
+- `homepage` is a property that sets the default HTML for the
+  `index.html` file with a default value of `'<h1>Hello world!</h1>'`
+- the `action` block uses the built-in collection of resources to tell
```

Comment:
```suggestion
- The `action` block uses the built-in collection of resources to tell
```

Reply:
These aren't full sentences, so don't need capitalising in this case.